lib/i18n/backend/base.rb

```diff
@@ -253,7 +253,7 @@ def load_yml(filename)
         # toplevel keys.
         def load_json(filename)
           begin
-            ::JSON.parse(File.read(filename))
+            ::JSON.parse(File.read(filename), symbolize_names: true, freeze: true)
```

Comment:
@paarthmadan you could do the same than for Psych, check for `JSON.respond_to?(:load_file)`, it's a method that was added recently, a bit after `freeze`, and it will also give you bootsnap caching https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md#190

Reply:
Good point, added 👍